tests/integration/tests.mk

```diff
@@ -120,6 +120,21 @@ test.integration.race.kube: | $(JUNIT_REPORT)
 	${_INTEGRATION_TEST_WORK_DIR_FLAG} \
 	2>&1 | tee >($(JUNIT_REPORT) > $(JUNIT_UNIT_TEST_XML))
 
+# Integration tests that detect race condition for native environment.
+.PHONY: test.integration.race.native
+test.integration.race.native: | $(JUNIT_REPORT)
+	mkdir -p $(dir $(JUNIT_UNIT_TEST_XML))
+	set -o pipefail; \
+	$(GO) test -race -p 1 ${T} ${TEST_PACKAGES} --log_output_level=CI:debug,tf:debug -timeout 120m \
```

Comment:
How long does this actually take? 120m seems pretty long.

Also not sure why log level is set?

Reply:
Based on @utka comments at https://github.com/istio/test-infra/pull/1215, these tests are added in postsubmit first. According to [golang document](https://blog.golang.org/race-detector), race detection can use ten times the CPU and memory. In case the race condition detection also needs longer time to run, I enlarged the timeout value from 30 minutes to 120 minutes. The log level has been removed.

Reply:
The tests take 1-2 min currently which is why I thought it was a little high, 30 min was probably too high too. Not a big issue though